packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-scheme-type.ts

```diff
@@ -0,0 +1,28 @@
+import { SecuritySchemeElement } from '@swagger-api/apidom-ns-openapi-3-1';
+import { AnnotationElement } from '@swagger-api/apidom-core';
+
+type SecuritySchemeTypePluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const securitySchemeTypeRefractorPlugin =
+  ({ annotations }: SecuritySchemeTypePluginOptions) =>
+  () => ({
+    visitor: {
+      SecuritySchemeElement(element: SecuritySchemeElement) {
+        if (element.get('type')?.toValue() === 'mutualTLS') {
+          const annotation = new AnnotationElement(
+            '"mutualTLS" type is not supported in OpenAPI 3.0.3.',
+            { classes: ['warning'] },
```

Comment:
I would go for `error` level here. `mututalTLS` is not supported in OpenAPI 3.0.3. If we're just leaving the wrong value there (`mutualTLS`), then the tooling consuming the output of the converter will most probably fail. 

What is the overall strategy of handling these cases? 

```suggestion
            { classes: ['error'] },
```

Reply:
The strategy is determined case by case. I just checked again the design document (see: SWG-9731) and indeed it seems I should be mutating the value here (`Annotate and remove this type/leave empty string`) so I will change the implementation.